src/lib/components/Form/Form.js

```diff
@@ -43,6 +44,9 @@ const Form = ({
     const fieldConfigs = model?.applyFieldConfig ? model?.applyFieldConfig({ data, lookups }) : defaultFieldConfigs;
     let gridApi = `${url}${model.api || api}`
     const { mode } = stateData.dataForm;
+    const userData = stateData.getUserData;
+    const fallbackPermissions = {edit: permissions.edit,delete: permissions.delete};
```

Comment:
add permission is misssing. 
If someone start to use directly then it by default will come as false always.